editor/src/core/shared/github.ts

```diff
@@ -625,63 +765,138 @@ export function mergeProjectContents(
  *       \-->branch
  */
 export function mergeProjectContentsTree(
+  currentTime: number,
   fullPath: string,
   currentContents: ProjectContentsTree | null,
   originContents: ProjectContentsTree | null,
   branchContents: ProjectContentsTree | null,
-): ProjectContentsTree | null {
-  if (originContents == null && branchContents == null) {
-    // Origin and branch lack an entry, so go with whatever the Utopia project contains.
-    return currentContents
-  } else if (originContents == null && currentContents == null) {
-    // Origin and Utopia project lack an entry, so go with whatever the branch contains.
-    return branchContents
-  } else if (
+): WithTreeConflicts<ProjectContentsTree | null> {
+  if (
     isProjectContentFile(currentContents) &&
     isTextFile(currentContents.content) &&
     isProjectContentFile(originContents) &&
     isTextFile(originContents.content) &&
     isProjectContentFile(branchContents) &&
     isTextFile(branchContents.content)
   ) {
-    // All 3 branches are a file.
-    const mergedResult = mergeDiff3(
-      currentContents.content.fileContents.code,
-      originContents.content.fileContents.code,
-      branchContents.content.fileContents.code,
-      {
+    // At this path it was and in Utopia and in the branch is a text file.
```

Comment:
I think there's a typo here, or a missed word, or something...

Reply:
Fixed in fe952b182.